packages/mdc-textfield/mdc-textfield.scss

```diff
@@ -375,6 +349,16 @@ $mdc-textfield-disabled-border-on-dark: rgba(white, .3);
   }
 }
 
+.mdc-textfield--invalid {
+  .mdc-textfield__label {
+    color: $mdc-textfield-error-on-light;
```

Comment:
I believe they are the same color https://storage.googleapis.com/material-design/publish/material_v_11/assets/0B5ZSepuCX1xOb195LU9KWEdIeEk/states.png